Documentation/contributing.rst

```diff
@@ -812,303 +960,383 @@ Have a question about how the tests work or want to chat more about improving th
 
-11. Confirm that you want to disable the build trigger.
+#. Validate the ``release-note/*`` label and check the PR title for release
+   note suitability. Put yourself into the perspective of a future release
+   notes reader with lack of context and ensure the title is precise but brief.
 
-.. image:: images/cilium-quayio-tag-7.png
-    :width: 600px
-    :align: center
-    :height: 300px
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | Labels                            | When to set                                                               |
+   +===================================+===========================================================================+
+   | ``dont-merge/needs-release-note`` | Do NOT merge PR, needs a release note                                     |
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | ``release-note/bug``              | This is a non-trivial bugfix                                              |
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | ``release-note/major``            | This is a major feature addition                                          |
+   +-----------------------------------+---------------------------------------------------------------------------+
+   | ``release-note/minor``            | This is a minor feature addition                                          |
+   +-----------------------------------+---------------------------------------------------------------------------+
 
-12. Once the build is finished click under Tags (on the left menu).
-13. Click on the wheel and;
-14. Add a new tag to the image that was built.
+#. Check for upgrade compatibility impact and if in doubt, set the label
+   ``upgrade-impact`` and discuss in the Slack channel.
 
-.. image:: images/cilium-quayio-tag-8.png
-    :width: 600px
-    :align: center
-    :height: 300px
+   +--------------------------+---------------------------------------------------------------------------+
+   | Labels                   | When to set                                                               |
+   +==========================+===========================================================================+
+   | ``upgrade-impact``       | The code changes have a potential upgrade impact                          |
+   +--------------------------+---------------------------------------------------------------------------+
 
-15. Write the name of the tag that you want to give for the newly built image.
-16. Confirm the name is correct and click on "Create Tag".
+#. When everything looks OK, approve the changes.
 
-.. image:: images/cilium-quayio-tag-9.png
-    :width: 600px
-    :align: center
-    :height: 300px
+#. When all review objectives for all ``CODEOWNERS`` are met and all CI tests
```

Comment:
One thing to note - only members of the cilium GitHub organization can actually add labels.

Reply:
note added